src/ai/backend/manager/api/auth.py

```diff
@@ -566,9 +568,13 @@ async def _pipe_builder(r: Redis) -> RedisPipeline:
         # Populate the result to the per-request state dict.
         request.update(auth_result)
 
-    # No matter if authenticated or not, pass-through to the handler.
-    # (if it's required, auth_required decorator will handle the situation.)
-    return await handler(request)
+    with ExitStack() as stack:
+        user_id = request.get("user", {}).get("id")
+        if user_id is not None:
+            stack.enter_context(with_user_id(auth_result["user"]["uuid"]))
```

Comment:
In the below code, user_id is actually user email instead of user uuid.
`user_id = request.get("user", {}).get("id")`